Tests/CSharp/NamespaceLevelTests.cs

```diff
@@ -379,7 +379,7 @@ public void MyClassVirtualCallProperty()
     Overridable Property P1() As Integer = 1
     MustOverride Property P2() As Integer
     Public Sub TestMethod() 
-        Dim w = MyClass.P1
+        Dim w = MyClass.p1

```

Comment:
Was this intentional? This doesn't look right.

Reply:
The intent was to test that it still works even when it's been accessed with the "wrong" case in VB, I actually meant to add a comment to the test to say that - will do so

Reply:
My other comment got lost somehow, realised that afterwards. Looks fine :)